teams/5/settings.html

```diff
@@ -0,0 +1,290 @@
+  <link rel="stylesheet" href="settings.css">
+  <title>Settings</title>
+</head>
+
+
+
+<body>
+    <nav class="navbar">
+      <div class="first">
+        <p>nav</p>
+
+        <ul class="top">
+          <p>ul</p>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+        </ul> 
+
+        <ul class="bottom">
+          <p>ul</p>
+          <li><i>i</i><a href="#">ul li a</a></li>
+          <li><i>i</i><a href="#">ul li a</a></li>
+          <li>    
+            <img src="ms.jpg" alt="img" width="50px" height="50px">
+          </li>
+        </ul> 
+      </div>
+      <div class="second">
+        <ul>
+          <p>ul</p>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span> </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+   
+        </ul>  
+       
+         <ul>       
+          <p>ul</p>  
+          <li><div class="spaceTopTwo"><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+         <li>
+           <div class="row"></div>
+             <div  class="spaceTopTwo">
+             <h2>h2</h2>
+             <p>p</p>
+             </div>
+           <i>i</i>
+          </div>
+        </li> 
+        </ul>  
+      </div>
+    </nav> 
+
+ <main>
+    <div class="mainHead">
+     <p>main div</p>
+    </div>
+
+  <header>
+  
+  <div class="row">
+    <img src="ms.jpg" alt="img" width="150px" height="150px">
+
+   <div class="margin">
+      <p>header</p>
+      <span>span</span>
+       <span>span</span>
+    </div>
+
+    <div class="margin">
+      <button>Button</button>
+      <button>Button</button>
+    </div>
+
+
+    
+    </div>
+
+    
+
+  </header>
+
+
+
+  <section>
+
+
+ <div class="row"> 
+
+  <div class="left">
+     <h2>h2</h2>
+     <p>p</p>
+    </div>
+
+
+  <div class="right">
+      <form action="">
+        <div>
+        <label>form label</label>
+        <input type="text">
+        <label>label</label>
+        <input type="text">
+        </div>
+
+        <div>
+        <label>label</label>
+        <input type="text">
+        </div>
+
+      </form>
```

Comment:
is it correct to close the </form> here?

Reply:
what is correct?